src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,41 @@
+package org.example;
+
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class Main
+{
+    private static final Logger log = Logger.getLogger(Main.class);
+    private static final Logger orderLogger = Logger.getLogger("SECOND_LOGGER");
+
+    public static void main( String[] args )
+    {
+        Order order = new Order();
```

Comment:
Let's wrap this code into the loop to add ability to add more than 1 order during program execution